VisualStudio.gitignore

```diff
@@ -22,24 +24,37 @@ bld/
 [Bb]in/
 [Oo]bj/
 
-# Visual Studio 2015 cache/options directory
-.vs/
-
-# MSTest test Results
+# MSTest
+# a test framework from Microsoft
+# remove: if you are not using the MSTest runner to run your tests
```

Comment:
So @arcresu mentioned "actions" in the gist as a way to call out when rules can be ignored or removed. There was a proposal for `enable-if` and `remove-if` to annotate this, but after wrestling with a couple of examples I found myself settling on this format.

- I experimented with `-when` here as well as `-if` but this lead to me writing the context in a strange way. So I dropped it altogether to just focus on the words associated with `remove`.
- I don't yet have an example of the dual to this yet (a rule that you should enable conditionally) but initial feels on this make me think it should be `include` rather than `enable` - probably just me wrestling with semantics.